packages/clap/clap.0.1.0/opam

```diff
@@ -0,0 +1,16 @@
+opam-version: "2.0"
+maintainer: "Romain Bardou"
+authors: [ "Romain Bardou" ]
+homepage: "https://github.com/rbardou/clap"
+bug-reports: "https://github.com/rbardou/clap/issues"
+dev-repo: "git+https://github.com/rbardou/clap.git"
+license: "MIT"
```

Comment:
```suggestion
license: "MIT"

depends: [
   "ocaml" {>= "4.02"}
   "dune"
]
```

The package at least depends on dune. I would recommend to commit to the sources also your `dune-project` file

Reply:
Please make these changes also upstream, I believe dune >= 1.0 should be fine for what I saw.
I have put ocaml 4.02 as dependency to check if a lower bound on the ocaml version is needed

Reply:
Thank you! I'll upstream the changes.